src/utils/components/ApplyStorageProfileSettingsCheckbox/ApplyStorageProfileSettingsCheckbox.tsx

```diff
@@ -47,15 +36,12 @@ const ApplyStorageProfileSettingsCheckbox: React.FC<ApplyStorageProfileSettingsC
       isInline
     >
       <Checkbox
-        onChange={(checked) =>
-          dispatchDiskState({
-            payload: checked,
-            type: diskReducerActions.SET_APPLY_STORAGE_PROFILE_SETTINGS,
-          })
-        }
+        onChange={(checked) => {
+          handleChange(checked);
+        }}
```

Comment:
nit: onChange={handleChange} should work the same :)

Reply:
Done!